.travis.yml

```diff
@@ -17,14 +18,15 @@ before_script:
   # Bootstrap only xapian-core for emscripten build.
   - git clone https://github.com/xapian/xapian
   - cd xapian
+  - git checkout tags/v1.4.11
   - ./bootstrap xapian-core
-  - ./configure CXXFLAGS=-O0 --disable-backend-honey --disable-backend-inmemory --disable-backend-remote
+  - ./configure CXXFLAGS='-O0 -Wno-error=deprecated-copy' --disable-backend-honey --disable-backend-inmemory --disable-backend-remote
   - make -j2
   - make -j2 distclean
   - cd xapian-core
-  - docker run -v $(pwd):/src trzeci/emscripten emconfigure ./configure CPPFLAGS='-DFLINTLOCK_USE_FLOCK' CXXFLAGS='-Oz -s USE_ZLIB=1 -fno-rtti' --disable-backend-honey --disable-backend-inmemory --disable-shared --disable-backend-remote
-  - docker run -v $(pwd):/src trzeci/emscripten emmake make
+  - docker run -v $(pwd):/src trzeci/emscripten:sdk-tag-1.38.29-64bit emconfigure ./configure CPPFLAGS='-DFLINTLOCK_USE_FLOCK' CXXFLAGS='-Oz -s USE_ZLIB=1 -fno-rtti' --disable-backend-honey --disable-backend-inmemory --disable-shared --disable-backend-remote
```

Comment:
Instead of running with the docker image, better to install emscripten in the pipeline like this: https://github.com/petersalomonsen/wasm-git/blob/master/.travis.yml#L8

Reply:
Addressed in the rebased version of this :)